packages/common.vm/tools/vm.common/vm.common.psm1

```diff
@@ -482,7 +482,7 @@ function VM-Install-From-Zip {
     [string] $category,
     [Parameter(Mandatory=$true, Position=2)]
     [string] $zipUrl,
-    [Parameter(Mandatory=$true, Position=3)]
+    [Parameter(Mandatory=$false, Position=3)]
```

Comment:
Does this actually work? Seems like it's used down at like 514. Perhaps the empty/null value allows it to not use the checksum?

Reply:
You can see it working in the tests: https://github.com/mandiant/VM-Packages/runs/4585639925?check_suite_focus=true

Reply:
Neat! 📷